BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -489,7 +489,8 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     }
     
     func openGoal(_ goal: Goal) {
-        let goalViewController = GoalViewController(goal: goal)
+        let viewModel = GoalViewModel(goal: goal)
```

Comment:
Let's continue passing the `Goal` into `GoalViewController` and have it construct the `GoalViewModel` (see comment below for my reasons why)

Reply:
Though we should move away from passing around nsmanagedobjects directly anyway. I did not change that here as we see.